crates/proof-of-sql/src/base/database/column_operation.rs

```diff
@@ -398,6 +507,61 @@ where
     ))
 }
 
+/// Divide an owned column by another.
+///
+/// We do not check for length equality here
```

Comment:
Can you document the behavior that this gives. (i.e. rounding, precision, scale, divide by zero, etc.)
Can you also add a note of what this is intended to mirror comes from?

Reply:
Sure!